server/graphql/common/expenses.ts

```diff
@@ -747,6 +758,23 @@ async function validateExpensePayout2FALimit(req, host, expense, expensePaidAmou
   }
 }
 
+const validateExpenseCustomData = (value: Record<string, unknown> | null): void => {
+  if (!value) {
+    return;
+  }
+
+  // Validate type: must be a JSON object
+  if (typeof value !== 'object') {
+    throw new ValidationFailed('Expense custom data must be an object');
+  }
+
+  // Validate size
+  const payloadSize = Buffer.byteLength(JSON.stringify(value), 'utf8');
+  if (payloadSize > 10e3) {
+    throw new ValidationFailed(`Expense custom data cannot exceed 10kb. Current size: ${payloadSize / 1000}kb`);
```

Comment:
```suggestion
    throw new ValidationFailed(`Expense custom data cannot exceed 10kB. Current size: ${payloadSize / 1000}kB`);
```

A bit pedantic but here we go.

Reply:
https://en.wikipedia.org/wiki/Kilobyte